source/entities/MapLight.cpp

```diff
@@ -20,15 +20,12 @@
 #include "entities/Tile.h"
 #include "game/Seat.h"
 #include "gamemap/GameMap.h"
-#include "modes/ModeManager.h"
 #include "network/ODPacket.h"
 #include "network/ODServer.h"
 #include "network/ServerNotification.h"
 #include "render/RenderManager.h"
-#include "render/ODFrameListener.h"
-#include "utils/Helper.h"
 #include "utils/LogManager.h"
-#include "utils/Random.h"
+//#include "utils/Random.h"
```

Comment:
Np. This can wait for another PR or something. It's already good to see lights being functional again, even if not (yet) changing.
There is not much before release btw. :)